markdown/solace-k8s-operator/solace-k8s-operator.md

```diff
@@ -0,0 +1,214 @@
+author: Ken Barr
+summary: How to build a Solace PubSubPlus Kubernetes Operator
+id: solace-k8s-operator
+tags: iguide
+categories: Kubernetes
+environments: Web
+status: Draft
+feedback link:
```

Comment:
Is this where you want the readers to refer to for feedback https://github.com/SolaceProducts/pubsubplus-kubernetes-quickstart?

Reply:
Not sure, There really is not git repo, this is just a procedure.  I can set as the underlaying Helm repo for now